src/variables.css

```diff
@@ -0,0 +1,13 @@
+:root {
+  --white-color: #fff;
+  --turquoise-color: #00acdc;
+  --text-color1: #060b35;
+  --text-color2: #616070;
```

Comment:
It's great that you using variables for colors, but this two have bad names 🥲 Better to name it as `primary` and `secondary` colors than `1` and `2`